manual/cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/ShortestPathPlanningTest.scala

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright (c) 2002-2015 "Neo Technology,"
+ * Network Engine for Objects in Lund AB [http://neotechnology.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Neo4j is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.neo4j.cypher.docgen
+
+import org.neo4j.cypher.docgen.tooling._
+import org.neo4j.graphdb.Path
+
+class ShortestPathPlanningTest extends DocumentingTest {
+  override def outputPath = "target/docs/dev/execution-plan-groups/"
+  override def doc = new DocBuilder {
+    doc("Shortest path planning", "query-shortestpath-planning")
+    initQueries(
+      """CREATE (charlie:Person {name:'Charlie Sheen'}),
+        |       (martin:Person {name: 'Martin Sheen'}),
+        |       (michael:Person {name: 'Michael Douglas'}),
+        |       (oliver:Person {name: 'Oliver Stone'}),
+        |       (rob:Person {name: 'Rob Reiner'}),
+        |
+        |       (wallStreet:Movie {title: 'Wall Street'}),
+        |       (charlie)-[:ACTED_IN {role: "Bud Fox"}]->(wallStreet),
+        |       (martin)-[:ACTED_IN {role: "Carl Fox"}]->(wallStreet),
+        |       (michael)-[:ACTED_IN {role: "Gordon Gekko"}]->(wallStreet),
+        |       (oliver)-[:DIRECTED]->(wallStreet),
+        |
+        |       (thePresident:Movie {title: 'The American President'}),
+        |       (martin)-[:ACTED_IN {role: "A.J. MacInerney"}]->(thePresident),
+        |       (michael)-[:ACTED_IN {role: "President Andrew Shepherd"}]->(thePresident),
+        |       (rob)-[:DIRECTED]->(thePresident)""",
+        "CREATE INDEX ON :Person(name)"
+    )
+    synopsis("Shortest path finding in Cypher and how it is planned")
+    section("-=-=-=-=-=-=-=-") {
+      p( """Planning shortest paths in Cypher can lead to different query plans depending on the predicates that need
+        |to be evaluated. Internally, Neo4j will use a fast bidirectional breadth-first search algorithm if the
+        |predicates can be evaluated whilst searching for the path. If the predicates need to inspect the whole path
+        |before deciding on whether it is valid or not, this fast algorithm cannot be used, and Neo4j will fall back to
+        |a slower exhaustive search for the path. The difference between these two can be in the order of magnitude,
```

Comment:
Is that how you say it `in the order of magnitude`? I've only ever heard `with several order of magnitudes`-style sentences. Perhaps `The difference between these two can be several orders of magnitude`?